pkg/apis/numaflow/v1alpha1/pipeline_types.go

```diff
@@ -511,6 +511,9 @@ type PipelineSpec struct {
 	// InterStepBuffer configuration specific to this pipeline.
 	// +optional
 	InterStepBuffer *InterStepBuffer `json:"interStepBuffer,omitempty" protobuf:"bytes,9,opt,name=interStepBuffer"`
+	// ExactlyOnce is the exactly-once settings for the pipeline.
+	// +optional
+	ExactlyOnce *ExactlyOnce `json:"exactlyOnce,omitempty" protobuf:"bytes,10,opt,name=exactlyOnce"`
```

Comment:
Instead of calling it `exactlyOnce`, use `deliverySemantics`, and specify which one is used.